Content/posts/update-02-october-2020.md

```diff
@@ -0,0 +1,101 @@
+---
+date: 2020-10-14 14:34
+description: An update on what happened in the SwiftWasm ecosystem during the first half of October 2020.
+---
+# What's new in SwiftWasm #2
+
+Welcome to the second SwiftWasm update! To make the updates flow steady, we're trying to publish
+them fortnightly now. Let us know what you think of this new cadence. And here's a gentle reminder
+that this blog is fully open-source, so if you spot a typo, an error, a broken link, or have any
+other feedback, please feel free to file it [in our `blog.swiftwasm.org` GitHub
+repository](https://github.com/swiftwasm/blog.swiftwasm.org).
+
+## SwiftWasm Team Updates
+
+We would like to welcome [@yonihemi](https://github.com/yonihemi) to the SwiftWasm team who joined
+us in the beginning of October. After the previous contributions he made to
+[`carton`](https://carton.dev) it made perfect sense to expand our team. As always, we invite
+everyone to contribute to any of our repositories, and it doesn't require much prior experience with
+SwiftWasm if any at all. Bug fixes, feature additions, improved documentation and related changes
+are very much appreciated and allow our ecosystem to grow even more!
+
+## JavaScriptKit
+
+[JavaScriptKit](https://github.com/swiftwasm/JavaScriptKit) had a few important updates in October
+so far. Most importantly, now that [#91](https://github.com/swiftwasm/JavaScriptKit/pull/91) by
+[@kateinoigakukun](https://github.com/kateinoigakukun) was merged, JavaScriptKit no longer uses
+unsafe flags in its `Package.swift`. The use of unsafe flags was a big problem for us, as it breaks
+dependency resolution due to strict checks that SwiftPM applies. If any package in your dependency
+tree contains unsafe flags, you can no longer depend on its semantic version, or a semantic version
+of any other package that depends on it.
+
+So far we were able to work around that with a hardcoded check for JavaScriptKit in our fork of
+SwiftPM. This was obviously a very ugly hack, but the biggest downside of that approach was that
+you couldn't depend on any package that had a semantic version dependency on JavaScriptKit in
+upstream Swift toolchains. That meant that libraries like [Tokamak](https://tokamak.dev) could not
+be built for macOS or Linux. And while the WebAssembly DOM renderer in Tokamak is the most useful
+module right now, this prevented us from working static HTML rendering in Tokamak on macOS and
```

Comment:
`prevented us from working` ? Not sure I follow the actual meaning.

Reply:
Well spotted! Updated to "this prevented static HTML rendering from working on macOS and
Linux".